src/main/java/mate/model/Driver.java

```diff
@@ -37,21 +59,12 @@ public void setLicenseNumber(String licenseNumber) {
     }
 
     @Override
-    public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
-        Driver driver = (Driver) o;
-        return Objects.equals(id, driver.id)
-                && Objects.equals(name, driver.name)
-                && Objects.equals(licenseNumber, driver.licenseNumber);
-    }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(id, name, licenseNumber);
+    public String toString() {
+        return "Driver{"
+                + "id=" + id
+                + ", name='" + name + '\''
+                + ", licenseNumber='" + licenseNumber + '\''
+                + ", userName='" + login + '\''
+                + '}';
```

Comment:
our model classes are considered correctrly executed when they have override methods equals(), hashcode() and toString()